tools/Iwyu/boost-all.imp

```diff
@@ -5621,42 +5621,4 @@
     { include: ["<boost/xpressive/detail/utility/save_restore.hpp>", private, "<boost/xpressive/regex_algorithms.hpp>", public ] },
     { include: ["<boost/xpressive/detail/utility/sequence_stack.hpp>", private, "<boost/xpressive/match_results.hpp>", public ] },
     { include: ["<boost/xpressive/traits/detail/c_ctype.hpp>", private, "<boost/xpressive/traits/c_regex_traits.hpp>", public ] },
-    { include: ["<boost/functional/hash/hash.hpp>", private, "<boost/functional/hash.hpp>", public ] },
```

Comment:
Do I understand correctly that this file is now exactly https://github.com/include-what-you-use/include-what-you-use/blob/master/boost-all.imp? Could you add a comment at the top of this file saying that, or a README.md in the `/tools/Iwyu` directory referencing the IWYU repository? We should probably also mention IWYU's license. We keep other files from external repositories in `/external`, so this file could also move there, but then it would be separate from the custom `.imp` files. What do you think?

Reply:
- Hmm.. this is nearly the same as the iwyu `boost-all.imp` file. It differs by one line, and that line is (already was) in the custom mappings. So it's probably okay to replace by the original.

- This was released under the LLVM license. I'm not clear on whether the original license needs re-attaching — does this file count as "source code"? it's a list of paths generated by a given script...

Reply:
we should include the LLVM license if it doesn't fall under our license